Cargo.toml

```diff
@@ -94,7 +94,7 @@ test = false # no unit tests
 [[bin]]
 name = "rustup-init"
 path = "src/cli/main.rs"
-test = false # no unit tests
+test = true
```

Comment:
One final question, does this need to be set to `true` or is it sufficient to remove the line?

Reply:
Good point. Let me try to remove the line.

Reply:
Here is a suggestion. To be explicit that we welcome unit tests, and potentially it may help others to write more unit tests, maybe we should keep it?

Reply:
Most of our tests are integration tests because unit tests have a tendency to increase rigidity in APIs, so while I don't want to reject tests, I also don't feel the need to massively encourage them at the unit-test level.

Reply:
Sounds good

Reply:
Squashed into one commit and removed a line.